lib/components/user/monitored-trip/trip-status-rendering-strategies/upcoming-trip-renderer.js

```diff
@@ -33,16 +34,32 @@ export default function upcomingTripRenderer ({
     monitoredTrip.leadTimeInMinutes,
     'minutes'
   )
-  data.bodyText =
-    `Trip is due to begin at ${tripStartMoment.format(timeFormat)}. (realtime monitoring will begin at ${monitoringStartMoment.format(timeFormat)})`
+  data.bodyText = (
+    <FormattedMessage
+      id='components.TripStatusRenderers.upcoming.tripBegins'
+      values={{
+        monitoringStart: monitoringStartMoment.format(timeFormat),
+        tripStart: tripStartMoment.format(timeFormat)
+      }}
+    />
+  )
 
   // set heading text and whether to show the alerts based on whether the
   // minutes until the trip starts is within the trip's lead time
   if (minutesUntilTripStart > monitoredTrip.leadTimeInMinutes) {
     // trip is not yet within lead time
     data.panelBsStyle = undefined
-    data.headingText =
-      `Next trip starts on ${tripStartMoment.format(extendedDateTimeFormat)}.`
+    data.headingText = (
+      <FormattedMessage
+        id='components.TripStatusRenderers.upcoming.nextTripBegins'
+        values={{
+          // FIXME: pass a single date value and a single date/time format.
```

Comment:
Since we decided to inline the date/time formats, do the same for this message (update the message in the language files), remove the moment variable and just pass `data.matchingItinerary.startTime`.